packages/create-playwright/assets/examples/4-pom.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { Page, test, expect } from '@playwright/test';
+
+test.describe('TODO tests', () => {
+  let todoPage: TodoPage;
+
+  test.beforeEach(async ({ page }) => {
```

Comment:
I'd put `new TodoPage(page)` in every test - this reinforces the "each test is given a page".